packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -870,19 +870,36 @@ function findLocalizePositions(
   return positions;
 }
 
-async function loadLocaleData(path: string, optimize: boolean): Promise<string> {
+async function loadLocaleData(path: string, optimize: boolean, es5: boolean): Promise<string> {
   // The path is validated during option processing before the build starts
   const content = fs.readFileSync(path, 'utf8');
 
-  // NOTE: This can be removed once the locale data files are preprocessed in the framework
-  if (optimize) {
-    const result = await terserMangle(content, {
-      compress: true,
-      ecma: 5,
-    });
+  // Downlevel and optimize the data
+  const transformResult = await transformAsync(content, {
```

Comment:
We we don't need to downlevel, why not just call `terserMangle` instead?

https://github.com/angular/angular-cli/blob/6c8b0f131595edcc278e14a4723ff01fd4419279/packages/angular_devkit/build_angular/src/utils/process-bundle.ts#L129-L164

Reply:
Consistency mainly but this also ensures that the files are supported regardless of the locale data source ECMAScript version.  The locale data files can now target the latest version (esnext) and still work.  terser is also unnecessary for these files since they are already generated to be minimal; comments and whitespace are really the only items that remain.